src/Simplex/Chat.hs

```diff
@@ -7086,6 +7122,8 @@ chatCommandP =
       "/_set alias @" *> (APISetContactAlias <$> A.decimal <*> (A.space *> textP <|> pure "")),
       "/_set alias :" *> (APISetConnectionAlias <$> A.decimal <*> (A.space *> textP <|> pure "")),
       "/_set prefs @" *> (APISetContactPrefs <$> A.decimal <* A.space <*> jsonP),
+      "/_set theme user " *> (APISetUserUITheme <$> A.decimal <*> optional (A.space *> jsonP)),
+      "/_set theme " *> (APISetChatUITheme <$> chatRefP <*> optional (A.space *> jsonP)),
```

Comment:
`_set chat theme` / `_set user theme` for order-independent parsing and natural reading?